gitpod-io/gitpod
# Update go dependencies

## Description

Updates:
- github.com/containerd/containerd v1.6.8
- github.com/prometheus/client_golang v1.13.0
- google.golang.org/grpc v1.49.0
- google.golang.org/protobuf v1.28.1
- cloud.google.com/go/storage v1.27.0
- cloud.google.com/go v0.104.0
- github.com/grpc-ecosystem/grpc-gateway/v2 v2.11.3

## Related Issue(s)

Fixes #

## How to test
- Check pods start
- Open a workspace

## Release Notes

```release-note
NONE
```

## Documentation


## Werft options:

      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`
